code/datums/abilities/werewolf/werewolf_feast.dm

```diff
@@ -136,7 +136,7 @@
 				interrupt(INTERRUPT_ALWAYS)
 				return
 
-			if (HH.decomp_stage <= 2 && !(ismonkey(target) || target.bioHolder && target.bioHolder.HasEffect("monkey"))) // Can't farm monkeys.
+			if (HH.decomp_stage <= 2 && !(istype(target, /mob/living/carbon/human/npc/monkey))) // Can't farm NPC monkeys. Player monkeys are still able to be mauled. Also removed redundant bioholder check.
```

Comment:
Why not just `/mob/living/carbon/human/npc`, if the goal is to force them to interact with the crew to farm points (rather than, say, Father Jack and co.)